src/tests/functional/plugin/conformance/test_runner/conformance_infra/src/read_ir_test/read_ir.cpp

```diff
@@ -224,6 +226,76 @@ void ReadIRTest::SetUp() {
             }
         }
     }
+
+#ifdef ENABLE_CONFORMANCE_PGQL
+    // Updating data in runtime. Should be set before possible call of a first GTEST status
+    auto pgLink = this->GetPGLink();
+    if (pgLink) {
+#    if defined(__arm__) || defined(_M_ARM) || defined(__aarch64__) || defined(_M_ARM64)
```

Comment:
Looks like we can use second approach for any devices, but we should get rid of hardcoded device name. What about to add additional field as `device_architecture` instead of `pgLink->SetCustomField("targetDevice", "CPU_ARM", true);` in data base?

Reply:
done